@DRAMOptions/DRAMOptions.m

```diff
@@ -188,28 +188,3 @@
       
    end
 end
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
```

Comment:
All the changes in this file are due to switching the inheritance from
matlab.mixin.SetGet to matlab.mixin.CustomDisplay